lib/core/errors.js

```diff
@@ -88,6 +88,16 @@ class ContentLengthMismatchError extends UndiciError {
   }
 }
 
+class TrailerMismatchError extends UndiciError {
+  constructor (message) {
+    super(message)
+    Error.captureStackTrace(this, TrailerMismatchError)
```

Comment:
Is this really needed? Why?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error

Reply:
https://nodejs.org/api/errors.html#errors_error_capturestacktrace_targetobject_constructoropt

Reply:
But `UndiciError` extends `Error`, so isn't `.stack` generated automatically?

Reply:
I'm actually unsure why this is necessary but given the best practices I've been able to read it seems to be the correct way...

Reply:
See the example from MDN:

```js
class CustomError extends Error {
  constructor(foo = 'bar', ...params) {
    // Pass remaining arguments (including vendor specific ones) to parent constructor
    super(...params)

    // Maintains proper stack trace for where our error was thrown (only available on V8)
    if (Error.captureStackTrace) {
      Error.captureStackTrace(this, CustomError)
    }

    this.name = 'CustomError'
    // Custom debugging information
    this.foo = foo
    this.date = new Date()
  }
}
```